### PR TITLE
Update Twitch-ads scriptlets

### PR DESCRIPTION
**Pull request by ryanbr**
Re-sync with changes https://github.com/pixeltris/TwitchAdSolutions/commit/144bc99a619aff738030c7d7de1ed833d43c2514 & https://github.com/pixeltris/TwitchAdSolutions/commit/0863c6d74b98b5e8349382f508412a9005f50c57

https://github.com/pixeltris/TwitchAdSolutions/issues/276#issuecomment-2299463794

### CODE REVIEW

**Review by thypon — APPROVED** [review]

File: resources/vaft-ublock-origin.js [hunk 1]
```diff
@@ -852,5 +811,61 @@
             return realFetch.apply(this, arguments);
         };
     }
-    hookFetch();
+    function onContentLoaded() {
+        // This stops Twitch from pausing the player when in another tab and an ad shows.
+        // Taken from https://github.com/saucettv/VideoAdBlockForTwitch/blob/cefce9d2b565769c77e3666ac8234c3acfe20d83/chrome/content.js#L30
```
[the remark this answers is not in the document]
Comment (ShivanKaul) [reply resources/vaft-ublock-origin.js:816]: Looks like a false positive

**Comment by ryanbr** [comment]
@diracdeltas  is this looks okay

File: resources/vaft-ublock-origin.js [hunk 2]
```diff
@@ -62,186 +20,187 @@
...
-                                                    if (IsPlayerAutoQuality) {
-                                                        e.data.value = 'auto';
+                        adBlockDiv.style.display = 'none';
+                    } else if (e.data.key == 'PauseResumePlayer') {
+                        doTwitchPlayerTask(true, false, false, false, false);
+                    } else if (e.data.key == 'ForceChangeQuality') {
+                        //This is used to fix the bug where the video would freeze.
+                        try {
+                            //if (navigator.userAgent.toLowerCase().indexOf('firefox') == -1) {
+                                return;
```
Comment (diracdeltas) [inline resources/vaft-ublock-origin.js:96]: minor issue but we can just delete or comment out the rest of this block

**Review by diracdeltas — APPROVED** [review]
it has some quality issues but looks non-malicious. i'm assuming this is just copied from upstream? 

going forward @thypon can approve instead of me

**Comment by ryanbr** [comment]
Yeah this a direct copy from upstream

**Comment by ShivanKaul** [comment]
@ryanbr remember to run `npm run build` after changing resources.

**Pull request merged**